exec/memory/base/src/main/java/io/netty/buffer/DrillBuf.java

```diff
@@ -287,14 +310,12 @@ public ByteOrder order() {
 
   @Override
   public ByteBuf order(ByteOrder endianness) {
-    // if(endianness != ByteOrder.LITTLE_ENDIAN) throw new
-    // UnsupportedOperationException("Drill buffers only support little endian.");
```

Comment:
This code breaks netty even though it shouldn't. We've had it commented out for years. Just removing it now.